nuxeo-features/nuxeo-platform-imaging-tiling/nuxeo-platform-imaging-tiling/src/main/java/org/nuxeo/ecm/platform/pictures/tiles/service/GCTask.java

```diff
@@ -41,7 +43,8 @@
 
     public static long getGCIntervalInMinutes() {
         if (GC_INTERVAL == 0) {
-            GC_INTERVAL = Long.parseLong(PictureTilingComponent.getEnvValue(GCINTERVAL_KEY, Long.toString(10)));
+            PictureTilingComponent ptc = (PictureTilingComponent) Framework.getService(PictureTilingService.class);
+            GC_INTERVAL = Long.parseLong(ptc.getEnvValue(GCINTERVAL_KEY, Long.toString(10)));
```

Comment:
you might consider to follow your previous practice and hardcode the string instead of executing it in place

Reply:
Same remark as above